doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -144,11 +144,20 @@ To perform a FOTA update, complete the following steps:
    .. note::
       nRF Connect for Desktop does not currently support the FOTA process.
 
-   Use `nRF Connect Device Manager`_, `nRF Connect for Mobile`_, or `nRF Toolbox`_ to update your device with the new firmware.
+   Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
 
    a. Ensure that you can access the :file:`dfu_application.zip` archive from your phone or tablet.
-   #. Connect to the device with the mobile app.
-   #. Initiate the DFU process to transfer the image to the device.
+   #. In the mobile app scan and select the device to update.
+   #. Switch to the :guilabel:`Image` tab.
+   #. Tap the :guilabel:`SELECT FILE` button and select the archive.
+   #. Tap the :guilabel:`START` button.
+   #. Select mode in the dialog window and tap the :guilabel:`START` button to initiate the DFU process to transfer the image to the device.
+   #. Wait for the DFU to finish and then verify that the application works properly.
+
+   .. note::
+      FOTA updates when MCUboot is in the direct-xip mode are supported since:
+      * Version ``1.8`` of `nRF Connect Device Manager`_ on Android.
+      * Version ``1.4`` of `nRF Connect Device Manager`_ on iOS.
```

Comment:
```suggestion
      Support for FOTA updates with MCUboot in the direct-xip mode is available since the following versions of the `nRF Connect Device Manager`_ mobile app:
      * Version ``1.8``  on Android.
      * Version ``1.4`` on iOS.
```
Let me know in case the meaning is affected